src/Illuminate/Support/Carbon.php

```diff
@@ -4,9 +4,8 @@
 
 use Carbon\Carbon as BaseCarbon;
 use Carbon\CarbonImmutable as BaseCarbonImmutable;
-use Carbon\Exceptions\InvalidFormatException;
+use Carbon\Exceptions\Exception;
```

Comment:
doesn't work on older versions. we can't bump the min carbon version in laravel 8

Reply:
Pretty much sure `InvalidArgumentException` of Carbon should work now.

Reply:
Okay, it uses `\InvalidArgumentException`, so it works now across versions.